modules/search/instant-search/components/search-result-minimal.jsx

```diff
@@ -0,0 +1,214 @@
+			return POST_TYPE_TO_ICON_MAP[ fields.post_type ];
+		}
+
+		switch ( fields.post_type ) {
+			case 'page':
+				if ( hasVideo ) {
+					return <Gridicon icon="video" size={ iconSize } />;
+				} else if ( hasAudio ) {
+					return <Gridicon icon="audio" size={ iconSize } />;
+				}
+				return <Gridicon icon="pages" size={ iconSize } />;
+			default:
+				if ( hasVideo ) {
+					return <Gridicon icon="video" size={ iconSize } />;
+				} else if ( hasAudio ) {
+					return <Gridicon icon="audio" size={ iconSize } />;
+				} else if ( hasGallery ) {
+					return <Gridicon icon="image-multiple" size={ iconSize } />;
+				}
+		}
+		return null;
+	}
+
+	renderNoMatchingContent() {
+		const path = new URL( 'http://' + this.props.result.fields[ 'permalink.url.raw' ] ).pathname;
+		const tags = this.getTags();
+		const cats = this.getCategories();
+		const noTags = tags.length === 0 && cats.length === 0;
+		return (
+			<div className="jetpack-instant-search__result-minimal-content">
+				{ noTags && <div className="jetpack-instant-search__result-minimal-path">{ path }</div> }
+				{ tags.length !== 0 && (
+					<div className="jetpack-instant-search__result-minimal-tag">
+						{ tags.map( tag => (
+							<span>
+								<Gridicon icon="tag" size={ this.getIconSize() } />
+								{ tag }
+							</span>
+						) ) }
+					</div>
+				) }
+				{ cats.length !== 0 && (
+					<div className="jetpack-instant-search__result-minimal-cat">
+						{ cats.map( cat => (
+							<span>
+								<Gridicon icon="folder" size={ this.getIconSize() } />
+								{ cat }
+							</span>
+						) ) }
+					</div>
+				) }
+			</div>
+		);
+	}
+
+	renderMatchingContent() {
+		return (
+			<div
+				className="jetpack-instant-search__result-minimal-content"
+				//eslint-disable-next-line react/no-danger
+				dangerouslySetInnerHTML={ {
+					__html: this.props.result.highlight.content.join( ' ... ' ),
+				} }
+			/>
+		);
+	}
+
+	renderComments() {
+		return (
+			this.props.result.highlight.comments && (
+				<div className="jetpack-instant-search__result-minimal-comment">
+					<Gridicon icon="comment" size={ this.getIconSize() } />
+					<span
+						className="jetpack-instant-search__result-minimal-comment-span"
+						//eslint-disable-next-line react/no-danger
+						dangerouslySetInnerHTML={ {
+							__html: this.props.result.highlight.comments.join( ' ... ' ),
+						} }
+					/>
+				</div>
+			)
+		);
+	}
+
+	render() {
+		const { locale = 'en-US' } = this.props;
+		const { result_type, fields, highlight } = this.props.result;
+		if ( result_type !== 'post' ) {
+			return null;
+		}
+
+		const noMatchingContent = ! highlight.content || highlight.content[ 0 ] === '';
+		return (
+			<div className="jetpack-instant-search__result-minimal">
+				<span className="jetpack-instant-search__result-minimal-date">
+					{ new Date( fields.date.split( ' ' )[ 0 ] ).toLocaleDateString( locale, {
+						dateStyle: 'short',
```

Comment:
Instead of relying on only the locale to display the date, do you think we could pass the date format from `get_option( 'date_format' )` as well, to get exactly the format used on the site, that may not be the default one for that locale.

As an example, my site's locale is `en_US`, but the date format I specified under Settings > General is `F j, Y`, not the short `m/d/y` version currently picked up from the locale and `dateStyle` right now.

Reply:
I think it'll be tricky given that we don't load any date formatting libraries for the sake of minimizing our bundle size. I'll look into this and report back.

Reply:
Can we do the date formatting on the API side instead?

Reply:
If we can't do much right now, could we at least move away from this terrible `m/d/y` format :P Anything would be better. :)

Reply:
To close the loop on this, this is tracked in #13810